akka-projection-kafka/src/main/resources/reference.conf

```diff
@@ -0,0 +1,2 @@
+# time to wait before retrieving last saved offset
```

Comment:
add short comment about why/when this is needed (as an extra safety)

Reply:
:+1: